### PR TITLE
CRM-19472 - Export headers for relationships are in machine name format

### PR DESCRIPTION
**Pull request by sqweets**
---
- CRM-19472: Export headers for relationships are in "machine name" format
  https://issues.civicrm.org/jira/browse/CRM-19472

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/Export/BAO/Export.php [hunk 1]
```diff
@@ -2118,6 +2117,8 @@ public static function setHeaderRows($field, $headerRows, $sqlColumns, $query, $
       $headerRows[] = $field;
     }
 
+    self::manipulateHeaderRows($headerRows, $contactRelationshipTypes);
```
Comment (jitendrapurohit) [inline CRM/Export/BAO/Export.php]: QA'd the PR, works fine and corrects the header for relationships. Just thinking if we could avoid calling of this method for every field and instead move this call to a little upward immediately after [`$headerRows` of contact relationships are evaluated](https://github.com/civicrm/civicrm-core/blob/master/CRM/Export/BAO/Export.php#L2118) ?

**Comment by colemanw** [comment]
@sqweets could you please make the recommended change and we'll get this merged, thanks.

**Comment by sqweets** [comment]
The recommended change has been made.  That is a better spot, thanks.

**Comment by colemanw** [comment]
@civicrm-builder add to whitelist

**Pull request merged**